06-building-a-program-start-to-finish/setting-up-software-development-environments-(deployment)/Output/README.md

```diff
@@ -0,0 +1,8 @@
+#### Output
+- **Using**: Heroku
+- **Deliver**: live URLs to a `review` app, your `staging` & `production` apps
+
+#### Deliverable
+- `review` <https://m-invertedindex-staging-pr-9.herokuapp.com/>
+- `staging` <https://m-invertedindex-staging.herokuapp.com/>
+- `production` <https://m-invertedindex.herokuapp.com/>
```

Comment:
That's because I haven't yet merged develop into master in my Inverted Index repo, waiting for your approval of the class and test before I do that.